_posts/2020-02-03-ftp.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+current: post
+cover: FTP.jpg
+title: Why you shouldn't be using FTP in 2020
+date: 2020-02-01 17:00:00
+category: tutorials
+author: Fiona McAllister
+tags: [open-source]
+class: post-template
+---
+
+## Introduction
+
+For many services, it is not enough to establish a connection. There are instances where a developer might want to transfer files from one server to another. FTP (File Transfer Protocol) is one of many available file transfer protocols that are available to use for servers. It enables the transfer of files from one machine or server to another over the internet. FTP itself is just an extension of the TCP protocol and uses the same core principles to establish and maintain a connection. The problem with this is that FTP (and TCP) are now almost 50 years old. 
+
+Since the development of FTP and TCP in the 1970s, cybersecurity has become a growing concern. FTP does not encrypt data and by default, neither does TCP. This means that even if you use a username and password over FTP, your data can still be compromised. Here we'll underline why exactly FTP is insecure and alternatives that you can use to keep your data, especially your credentials, secure.
```

Comment:
The issue isn't really with TCP. HTTPS uses TCP too, and so does most communication. HTTP/3 (and QUIC) uses UDP, which is rather interesting, but you still need TLS or similar to do the encryption.